File: src/main/Java/boj_step7/boj2563.java
import java.io.*;
import java.util.StringTokenizer;

public class boj2563 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        boolean[][] paper = new boolean[100][100];
        int n = Integer.parseInt(reader.readLine());
        int count = 0;
        while (n-- > 0) {
            StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
            int x = Integer.parseInt(tokenizer.nextToken()) - 1;
            int y = Integer.parseInt(tokenizer.nextToken()) - 1;
            for (int i = x; i <= x + 9; i++)
                for (int j = y; j <= y + 9; j++) {
                    if (!paper[i][j]) count++;
                    paper[i][j] = true;
                }
        }
        writer.append(String.valueOf(count));
        writer.close();
    }
}

File: src/main/kotlin/boj2563.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val n = r.readLine().toInt()
    var result = 0
    val square = Array(100) { BooleanArray(100) }
    repeat(n) {
        val (x, y) = r.readLine().split(" ").map { it.toInt() }
        for (i in y until y + 10)
            for (j in x until x + 10) {
                if(square[i][j]) continue
                square[i][j] = true
                result++
            }
    }
    w.write(result.toString())
    w.close()
}